mock-etcd/Cargo.toml

```diff
@@ -12,7 +12,7 @@ categories = ["storage"]
 [dependencies]
 async-lock = "2.3.0"
 async-io = "1.6.0"
-env_logger = "0.8.1"
+env_logger = "0.9.3"
```

Comment:
Here, are the changes different from the version changes in etcd_client?

Reply:
This change is for compile problem. Old version has some broken dependency relationship.